src/common/components/Table/Table.tsx

```diff
@@ -5,23 +5,25 @@ import caretUp2Icon from '../../assets/icons/ico-caret-up2.svg'
 import Checkbox from '../Checkbox/Checkbox'
 
 export type TRow = {
-  [index: string]: string | number
+  [index: string]: any
```

Comment:
`React.ReactNode`type is good for accepting jsx content

Reply:
![image](https://user-images.githubusercontent.com/63484644/122304431-cb8b7f80-cf37-11eb-9d78-69c8e7c4e3c4.png)

@ngvtuan and @CrazyOptimist, as you can see, if we don't use any, then it will require to add all types that require to `Table `component.  that is why I added  to avoid the case that multi type definition used.

Any ideas on this, please?

Reply:
![image](https://user-images.githubusercontent.com/63484644/122304823-6f752b00-cf38-11eb-8dff-0ce428d4c645.png)

Reply:
@prodev1101 IMO whole table component should be rewritten, I would use react-table library which covers many needs and has types.

For now let it be `any` type just in this case, please leave `// eslint-disable-next-line` comment and add `// TODO: rewrite table component` comment

Reply:
And please use `any` only as exception. In `custom?: (value: any` it seems to be too complex to avoid it, but in TRow `any` still can be replaced with ReactNode

Reply:
Please change `[index: string]: any` to `[index: string]: React.ReactNode`, and second `any` is ok

Reply:
Now it seems to me that both `any` can be replaced with React.ReactNode type - please replace, and let us know if that won't resolve the issue

Reply:
No, it doesn't resolve both case. I think we will need to migrate the current `Table` component.

Reply:
@prodev1101 just tried to fix it - agree with you, let's leave any for now